Libraries/Geolocation/RCTLocationObserver.m

```diff
@@ -32,6 +32,7 @@ typedef NS_ENUM(NSInteger, RCTPositionErrorCode) {
   double maximumAge;
   double accuracy;
   double distanceFilter;
+  bool useSignificantChanges;
```

Comment:
Use `BOOL` instead. There's some really subtleties with `bool` so we try to be consistent.